client.go

```diff
@@ -1264,7 +1289,9 @@ func (l *link) muxReceive(fr performTransfer) error {
 	// discard message if it's been aborted
 	if fr.Aborted {
 		l.buf.reset()
-		l.msg = Message{}
+		l.msg = Message{
+			doneSignal: make(chan struct{}),
```

Comment:
init the signal channel to avoid nil. Probably unecessary, since we take care of it in the Receive()